src/GUI/RevEng.Shared/RevEng.Common.csproj

```diff
@@ -5,6 +5,7 @@
     <NoWarn>$(NoWarn);EF1001</NoWarn>
     <SignAssembly>true</SignAssembly>
     <AssemblyOriginatorKeyFile>Key.snk</AssemblyOriginatorKeyFile>
+    <LangVersion>default</LangVersion>
```

Comment:
Why

Reply:
due to following code 
ObjectType objectType = instance switch
            {
                View _ => ObjectType.View,
                Function _ => ObjectType.ScalarFunction,
                StoredProcedure _ => ObjectType.Procedure,
                _ => ObjectType.Table,
            };